flows/knack/knack_services.py

```diff
@@ -0,0 +1,400 @@
+#!/usr/bin/env python
+
+"""
+Name: ATD Knack Services
+Description: This set of tasks publishes data from ATD's various Knack 
+ applications to other platforms including a Postgres database, Socrata, 
+ and ArcGIS Online (AGOL). 
+Schedule: "30 14,18 * * *"
+Labels: atd-data02, knack
+"""
+
+import os
+
+import docker
+import prefect
+from datetime import datetime, timedelta
+
+# Prefect
+from prefect import Flow, task, Parameter, unmapped, case
+from prefect.storage import GitHub
+from prefect.run_configs import LocalRun
+from prefect.engine.state import Failed
+from prefect.schedules import Schedule
+from prefect.schedules.clocks import CronClock
+from prefect.backend import set_key_value, get_key_value
+from prefect.triggers import all_successful
+from prefect.tasks.docker import PullImage
+
+
+from prefect.utilities.notifications import slack_notifier
+
+
+# Default parameters is for Signs/Markings Contractor Work Orders
+FLOW_NAME_UNIQUE = "Markings Contractor Work Orders"
+APP_NAME = "signs-markings"
+CONTAINER = "view_3628"
+LAYER_NAME = "markings_contractor_work_orders"
+# Parameter that will overwrite all data (ignores date)
+REPLACE_DATA = False
+APP_NAME_DEST = ""
+SOCRATA_FLAG = True
+AGOL_FLAG = True
+
+# Define current environment
+current_environment = "prod"
```

Comment:
I think this variable is just used once, and it's used to build the name of the knack services. Maybe just define that end result variable here .. the idea being to more easily reveal how this string will get used? I thought changing this value would effect more behaviors and what-have-you about what the flow would do based on its name.

Reply:
feels like a holdover from when we were going to have production flows and stagings flows running. The flow can just be named `knack_services_production` instead of using this variable too. 

Unless theres a chance we will have staging flows for knack services?

Reply:
I'm going to go ahead and zap this `current_environment` variable and just change the name to `knack_services_production`. I like @chiaberry's suggestion of using parameters for testing new code without having to go and re-register your flow with a different `docker_env`.